libs/langchain/langchain/chat_models/anyscale.py

```diff
@@ -61,7 +62,7 @@ def lc_secrets(self) -> dict[str, str]:
     anyscale_proxy: str | None = None
     available_models: set[str] | None = None
 
-    @root_validator()
+    @root_validator(pre=True)
```

Comment:
I saw in CI tests that reuse of the root validator function name was incorrect, so I changed it, and made this one run before the one it inherits from ChatOpenAI. I did some testing to be sure there were no collisions if OPENAI_API_KEY is already set as an environment variable for use with ChatOpenAI or similar.